azurerm/provider.go

```diff
@@ -163,6 +163,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_sql_elasticpool":                    resourceArmSqlElasticPool(),
 			"azurerm_sql_firewall_rule":                  resourceArmSqlFirewallRule(),
 			"azurerm_sql_server":                         resourceArmSqlServer(),
+			"azurerm_sql_administrator":                  resourceArmSqlAdministrator(),
```

Comment:
_minor_ since this is an `active directory administrator` could we make this `azurerm_sql_active_directory_administrator`?

Reply:
renamed as requested